script/pull-l10n.py

```diff
@@ -23,6 +25,14 @@ def parse_args():
     parser = argparse.ArgumentParser(description='Pull strings from Transifex')
```

Comment:
or Crowdin?

Reply:
Fixed in [13140e2](https://github.com/brave/brave-core/pull/27005/commits/13140e28ea7cd53e4056067f06938b892d409c16)